src/util/PortfolioUploadMediaService.js

```diff
@@ -0,0 +1,159 @@
+import {
```

Comment:
Nice use of a custom hook to handle this logic. 
Stretch goal: This is a pretty big file. Can we refactor some of the code to remove some repetition? There are a lot of similarities with the request logic and error handling logic.

Reply:
I tried really hard to reduce repetition where I could by combining uploading/deleting videos and photos for a portfolio. I'll try to come back to this a little later, when I finish another feature.

Reply:
Don't worry about this took much right now. It's crunch time and your application is working as it should. Take this as more of a reminder when you are working on your next codebase.